client/components/fields/headline.tsx

```diff
@@ -1,9 +1,12 @@
 import PropTypes from 'prop-types';
+import {getTranslatedValue} from '.';
 
-export const headline = ({item}) => item.headline || null;
+export const headline = ({item, filterLanguage}) => getTranslatedValue(filterLanguage, item, 'headline') ||
```

Comment:
When modifying existing functions, it would be good to add typescript types. We have a lot of code from when the source was javascript, so adding types when we modify the code is one way to add types everywhere (eventually)

Reply:
yep